section23_email_and_datetime/projects/automated_birthday_wisher/data_file_management.py

```diff
@@ -0,0 +1,77 @@
+import pandas
+
+class DataFileManagement():
+
+    def __init__(self):
+        pass
+
+
+    def print_invalid_input_with_y_and_n(self):
```

Comment:
We should throw and error if user inputs the invalid argument and better to do like this. So we will have just one method interacting with the user input

```
def get_input(self):
    input = input("Do you want to add some data to your file? (y/n): ")
    if input != 'y' or input != 'n':
         print("Invalid input!!!. Only 'y' and 'n' are allowed.")
         raise ValueError()
    if is_adding_data == 'y':
        return True
    elif is_adding_data == 'n':
        return False
```